unit_test/tstAdaptiveMesh.hpp

```diff
@@ -92,63 +91,6 @@ void constructionTest()
                            local_mesh_1.lowCorner( Cabana::Grid::Ghost(), 2 ) +
                                k * cell_size );
             }
-
```

Comment:
Why was the test for `mesh_2` removed?

Reply:
Almost certainly not intended to push. Added back